themes/ddbasic/template.php

```diff
@@ -34,8 +34,11 @@ function ddbasic_preprocess_html(&$vars) {
       $vars['classes_array'][] = 'search-form-extended';
       $vars['classes_array'][] = 'show-secondary-menu';
 
-      if (menu_get_item()['path'] === 'search/ting/%') {
-        $vars['classes_array'][] = 'extended-search-is-open';
+      $path = menu_get_item()['path'];
+      switch ($path) {
+        case 'search/ting/%';
+        case 'ding_frontpage':
```

Comment:
Adding `ding_frontpage` seems to be a change which is not related to the issue at hand. Is that intentional?

Reply:
Yes, as I remember this fixes som issues where you have selected the search field to default be open.